roles/self_hosted_first_host/defaults/main.yml

```diff
@@ -17,6 +17,7 @@ hosted_storage_path: /path/to/storage
 hosted_storage_lun: none:None
 hosted_iscsi_portal_port: 3260
 hosted_iscsi_portal_ip_address: '{{ hosted_storage_address }}'
+vm_timezone: America/New_York
```

Comment:
I wonder if there's a good way to get the default timezone from the ansible facts or something rather than defaulting to east.. Other than that looks good, no other comments. Comfortable just merging if you prefer.